.github/workflows/ci.yml

```diff
@@ -266,6 +267,9 @@ jobs:
           - target: x86_64-unknown-linux-gnu
             host_os: ubuntu-22.04
 
+          - target: s390x-unknown-linux-gnu
+            host_os: ubuntu-22.04
```

Comment:
In the analogous PPC64LE PR I just merged, the author noted that they kept the alphabetical order intact. Could you please do the same throughout this PR? It's an arbitrary rule but it avoids people needing to wonder where to put things.

Reply:
Used alphabetic order everywhere now.